### PR TITLE
BB-362 remove kafka connect connector config from logs

### PR DESCRIPTION
**Pull request by welansari**
kafka connect connector config contains the mongodb connection string which should not be logged

also fixing a bug with the oplog populator where if a connector update fails cause the connector can't be found, it is not considered an error

### CODE REVIEW

File: lib/wrappers/KafkaConnectWrapper.js [hunk 1]
```diff
@@ -242,18 +241,16 @@ class KafkaConnectWrapper {
...
                 throw err;
             }
         }
-        // do nothing if connector not found
         this._logger.error('Can\'t update connector config, connector not found', {
             method: 'KafkaConnectManager.updateConnectorConfig',
             connector: connectorName,
         });
-        return undefined;
+        throw errors.InternalError.customizeDescription('Could not find connector to update');
```
Comment (francoisferrand) [inline lib/wrappers/KafkaConnectWrapper.js:253]: not the same commit : this is unrelated to the logs...

**Review by francoisferrand — COMMENTED** [review]
.

**Review by KillianG — APPROVED** [review]

**Review by francoisferrand — APPROVED** [review]
